api/server/volume.go

```diff
@@ -598,5 +697,9 @@ func (vd *volAPI) Routes() []*Route {
 		{verb: "POST", path: snapPath("", volume.APIVersion), fn: vd.snap},
 		{verb: "GET", path: snapPath("", volume.APIVersion), fn: vd.snapEnumerate},
 		{verb: "POST", path: snapPath("/restore/{id}", volume.APIVersion), fn: vd.restore},
+		{verb: "GET", path: credsPath("/credslist", volume.APIVersion), fn: vd.credsList},
+		{verb: "POST", path: credsPath("/credscreate", volume.APIVersion), fn: vd.credsCreate},
+		{verb: "DELETE", path: credsPath("/credsdelete", volume.APIVersion), fn: vd.credsDelete},
```

Comment:
since the verb is DELETE, no need for "/credsdelete".

Reply:
OK. will do this.